Pursuit-Core-iOS-Episodes-from-Online/Pursuit-Core-iOS-Episodes-from-Online/Controllers/ViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  ViewController.swift
+//  Pursuit-Core-iOS-Episodes-from-Online
+//
+//  Created by Benjamin Stone on 9/5/19.
+//  Copyright © 2019 Benjamin Stone. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+        
+        @IBOutlet weak var showTableVIew: UITableView!
+        @IBOutlet weak var showSearchBar: UISearchBar!
+        @IBOutlet weak var spinnerOne: UIActivityIndicatorView!
+        
+        var viewShow = [TVShowsWrapper](){
+            didSet {
+                showTableVIew.reloadData()
+            }
+        }
+        
+        override func viewDidLoad() {
+            super.viewDidLoad()
+            showTableVIew.delegate = self
+            showTableVIew.dataSource = self
+            showSearchBar.delegate = self
+            loadData(word: nil)
```

Comment:
I like that you out a default loadData so that the tableview wouldn't be empty when the app loads for the first time. Good job on that.